SQL Server REAMDE docs (#128)
hasura/ndc-sqlserver: readme.md

# SQL Server Connector

[![Docs](https://img.shields.io/badge/docs-v3.x-brightgreen.svg?style=flat)](https://hasura.io/docs/3.0/getting-started/overview/)
[![ndc-hub](https://img.shields.io/badge/ndc--hub-sqlserver-blue.svg?style=flat)](https://hasura.io/connectors/sqlserver)
[![License](https://img.shields.io/badge/license-Apache--2.0-purple.svg?style=flat)](LICENSE.txt)
[![Status](https://img.shields.io/badge/status-alpha-yellow.svg?style=flat)](./readme.md)

> The Hasura SQLServer Connector allows for connecting to a SQL Server database giving you an instant GraphQL API on top

Reviewer: With this connector, Hasura allows you to instantly create a real-time GraphQL API on top of your data models in Microsoft SQL Server. This connector supports SQL Server's functionalities listed in the table below, allowing for efficient and scalable data operations. Additionally, users benefit from all the powerful features of Hasura’s Data Delivery Network (DDN) platform, including query pushdown capabilities that delegate query operations to the database, thereby enhancing query optimization and performance.

Author: Updated

> of your SQL Server data.

Reviewer: ```suggestion
```

| Filter / Search                 |    ✅     |       |
| Simple Aggregation              |    ✅     |       |
| Sort                            |    ✅     |       |
| Paginate                        |    ✅     |       |
| Table Relationships             |    ✅     |       |
| Views                           |    ✅     |       |
| Remote Relationships            |    ✅     |       |
| Custom Fields                   |    ❌     |       |
| Mutations                       |    ❌     |       |
| Distinct                        |    ✅     |       |
| Enums                           |    ❌     |       |
| Naming Conventions              |    ❌     |       |
| Default Values                  |    ❌     |       |
| User-defined Functions          |    ❌     |       |

## Before you get Started

1. Create a [Hasura Cloud account](https://console.hasura.io)
2. Install the [CLI](https://hasura.io/docs/3.0/cli/installation/)
3. Install the [Hasura VS Code extension](https://marketplace.visualstudio.com/items?itemName=HasuraHQ.hasura)
4. [Create a project](https://hasura.io/docs/3.0/getting-started/create-a-project)

## Using the connector

> To use the SQL server connector, follow these steps in a Hasura project:

Reviewer: ```suggestion
To use the SQL Server connector, follow these steps in a Hasura project:
```

   ```

   This will return information about the build:

   |               |                                                                                                   |
   | ------------- | ------------------------------------------------------------------------------------------------- |
   | Build Version | bd96bb221a                                                                                        |
   | API URL       | https://<PROJECT_NAME>-bd96bb221a.ddn.hasura.app/graphql                                          |
   | Console URL   | https://console.hasura.io/project/<PROJECT_NAME>/environment/default/build/bd96bb221a/graphql     |
   | Project Name  | <PROJECT_NAME>                                                                                    |
   | Description   |                                                                                                   |

   Follow the project configuration build [guide](https://hasura.io/docs/3.0/project-configuration/builds/) to apply
   other actions on the build.

5. Test the API

   The console URL in the build information cna be used to open the GraphiQL console to test out the API

## Contributing

> ndc-sqlserver is still in early stages of development and we are currently not accepting contributions.

Reviewer: Why?? We should accept all contributions right @gneeri ?

Reviewer 2: Yeah, I think if someone were willing to put in the time, we'd be happy for the contribution. Perhaps we could just say something like:

`ndc-sqlserver is still in early stages and we have some work actively underway, but we are always happy for any community contributions.`

## License

> The Hasura SQLServer connector is available under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0).

Reviewer: ```suggestion
The Hasura SQL Server connector is available under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0).
```

## Running the connector locally

To run the SQL server connector locally, follow these steps:

> 1. Initialize a configuration:

Reviewer: ```suggestion
1. Set up a connector configuration file:
```
